app/views/includes/assessmentHeading.scala.html

```diff
@@ -6,12 +6,12 @@
 <div id="assessment-heading">
     <ul id="assessment-heading" class="govuk-list">
         <li id="local-authority-reference">
-            <span class="govuk-body-s govuk-!-font-weight-bold">@messages("assessments.localAuthorityRef")</span>
-            <span class="govuk-body-s" id="local-authority-reference-code">@localAuthorityReference</span>
+            <span class="govuk-body-m govuk-!-font-weight-bold">@messages("assessments.localAuthorityRef")</span>
```

Comment:
This change is outside the scope of the Jira and will affect all pages. Are you bringing `assessmentHeading` inline with other services here?

Reply:
I made font size bigger, following prototype. I know is affecting all the views in PL, however, recently  I did https://jira.tools.tax.service.gov.uk/browse/VTCCA-4471, where the task was to increase font size across the whole check journey.